zebra-rpc/src/methods/tests/snapshot/snapshots/z_validate_address_basic@mainnet_10.snap.new

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
These snapshot files need a different name, there's two ways to fix this:
1. Take the ".new" off the end manually
2. Run the full `cargo insta` command again, and accept the changes interactively at the end (it will do the renames for you)

Reply:
Hm, I ran `cargo insta test --features getblocktemplate-rpcs`, but the filenames didn't change.

Reply:
Done in [c984d4d](https://github.com/ZcashFoundation/zebra/pull/6185/commits/c984d4d4c506558665b7e40a14d33b309d573bf3).